### PR TITLE
[skip ci] Nightly autorun script for the cronjob

### PR DESCRIPTION
**Pull request by rajanashok**
Nightly autorun script to use for the cronjob.

### CODE REVIEW

File: tests/nightly/nightly-autorun.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Get the latest code from vmware/vic repo
```
Comment (anchal-agrawal) [inline tests/nightly/nightly-autorun.sh]: This file needs the copyright header.

Comment (rajanashok) [inline tests/nightly/nightly-autorun.sh]: Updated now. thanks.

File: tests/nightly/nightly-autorun.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Get the latest code from vmware/vic repo
+cd ~/go/src/github.com/vmware/vic
+git checkout master
+git remote update
```
Comment (anchal-agrawal) [inline tests/nightly/nightly-autorun.sh]: Are the remotes already set up?

**Comment by mhagen-vmware** [comment]
lgtm

File: tests/nightly/nightly-autorun.sh (hunk 2 above)
Comment (rajanashok) [inline tests/nightly/nightly-autorun.sh]: yep

**Comment by anchal-agrawal** [comment]
:shipit:

**Pull request merged**